tsconfig.json

```diff
@@ -27,6 +27,8 @@
     "acceptance-tests",
     "webpack",
     "jest",
-    "src/setupTests.ts"
+    "src/setupTests.ts",
+    "src/**/*.test.ts",
```

Comment:
I've added these because the typescript compiler was not happy with me using `import` for my Jest mocks.  Using `require` was fine.